okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -144,15 +144,14 @@
   private final int initialWindowSize;
   private Listener listener;
   private FrameReader testFrameReader;
-  private AsyncFrameWriter frameWriter;
+  private ExceptionHandlingFrameWriter frameWriter;
```

Comment:
Does this now need to be GuardedBy("lock")? AsyncFrameWriter was thread-safe whereas the new implementation is not.